joomla/joomla-cms
# Smart search parser leaking CSS and JS terms

Pull Request for Issue #12408 .
### Summary of Changes

Strip head first because head may contain script or style blocks.
### Testing Instructions

Copy-paste an entire web page with a head block that contains inline CSS and/or JavaScript into a new article.  You might see CSS or JS terms in the autosuggester.  If you don't, try another web page until you do.  You can try this page: https://www.joomla.org/announcements/release-news/5418-joomla-254-released.html then search for "col2".

Apply this PR.

Copy-paste the same article again so it gets re-indexed.  The CSS/JS terms should no longer appear in the autosuggester.
### Documentation Changes Required

None.
